Session/Conversations/ConversationViewModel.m

```diff
@@ -830,6 +830,17 @@ - (void)updateViewWithOldItemIdList:(NSArray<NSString *> *)oldItemIdList
                 [updatedNeighborItemSet addObject:itemId];
             }
         }
+        
+        // Add the following item of a deleted message to update
+        // to show the date header of the deleted message if needed
+        for (NSString *deletedItemId in deletedItemIdSet) {
```

Comment:
Is this still needed even with the recent message updating fix?

Reply:
> Is this still needed even with the recent message updating fix?

Yeah, I think so. With the MessageInvalidator, the updated items won't include the neighbours but only the deleted message itself.